src/Sulu/Bundle/HttpCacheBundle/DependencyInjection/SuluHttpCacheExtension.php

```diff
@@ -14,17 +14,32 @@
 use Symfony\Component\Config\Definition\Exception\InvalidConfigurationException;
 use Symfony\Component\Config\FileLocator;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
 use Symfony\Component\DependencyInjection\Loader;
 use Symfony\Component\HttpKernel\DependencyInjection\Extension;
 
-class SuluHttpCacheExtension extends Extension
+/**
+ * Container extension for sulu-http-cache bundle.
+ */
+class SuluHttpCacheExtension extends Extension implements PrependExtensionInterface
 {
     /**
      * {@inheritdoc}
      */
-    public function getConfiguration(array $config, ContainerBuilder $container)
+    public function prepend(ContainerBuilder $container)
     {
-        return new Configuration();
+        if ('prod' !== $container->getParameter('kernel.environment')) {
```

Comment:
See https://github.com/sulu/sulu-standard/pull/760#issuecomment-259070615: I would check if the environment is `dev` or `test`, since in `staging` caching should be activated, to be as close as possible to the real environment.